src/validations/class.validation.js

```diff
@@ -33,16 +32,9 @@ const deleteClass = {
   })
 };
 
-const getAllClasses = {
-  params: Joi.object().keys({
-    userId: Joi.string().custom(objectId)
-  })
-};
-
 module.exports = {
   createClass,
   getClass,
   updateClass,
-  deleteClass,
```

Comment:
why you delete these two functions here?

Reply:
oh shit i didn't mean to delete `deleteClass` thanks for noticing, i just wanted to delete getAllClasses, since we don't need any validation for that

Reply:
Nvm i did not delete the first one